circle.yml

```diff
@@ -5,6 +5,10 @@ general:
 machine:
   xcode:
     version: "7.3"
+  environment:
+    GOPATH: "$HOME/go"
+    PROJECT_ROOT: "$GOPATH/src/github.com/docker/hyperkit"
+    PATH: "${PATH}:${GOROOT}/bin:${GOPATH}/bin"
```

Comment:
Is `${GOROOT}` not in some default location? Does circleci provide `${GOROOT} for us?